src/driverApp/tests/driver.test.js

```diff
@@ -21,14 +21,14 @@ describe('driver router tests', () => {
 	let token;
 	let createdDriverId;
 	let driverInfo;
-	it('should login an admin user', async () => {
+	it('should login an operator user', async () => {
 		const password = await bcrypt.hash('Andela@1234', 5);
```

Comment:
use the password created in dummy data file

Reply:
I just fixed that issue in the latest push. Take a look